src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift

```diff
@@ -207,7 +207,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 		updateExposureState(state)
 		Analytics.triggerAnalyticsSubmission()
 		appUpdateChecker.checkAppVersionDialog(for: window?.rootViewController)
-		healthCertificateService.updateDCCWalletInfosIfNeeded()
+		healthCertificateService.updateValidityStatesAndNotifications(completion: { })
```

Comment:
That technically correct, for sure. :)
However I would prefer an optional `completion` argument that can accept `nil`, instead an empty closure. ;)

Reply:
Yes but previously it's defined like this and they call the empty closure in various point.
If I used nil then it will shows the error: "'nil' is not compatible with expected argument type '() -> Void'"

Reply:
If you make the completion argument optional, it will accept nil too.

Reply:
Okay, if empty closure were fine before, we're gonna keep that on your code.